src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceImplIntTestGetOffenderIdentifiers.kt

```diff
@@ -0,0 +1,113 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.junit.jupiter.api.Test
+import org.mockito.ArgumentMatchers
+import org.mockito.kotlin.whenever
+import org.springframework.boot.test.mock.mockito.MockBean
+import uk.gov.justice.hmpps.prison.repository.jpa.model.Offender
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderBooking
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderIdentifier
+import uk.gov.justice.hmpps.prison.repository.jpa.model.Staff
+import uk.gov.justice.hmpps.prison.repository.jpa.model.StaffUserAccount
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderIdentifierRepository
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+class OffenderResourceImplIntTestGetOffenderIdentifiers : ResourceTest() {
+  private val staffUserAccount = StaffUserAccount.builder().username("johnsmith").staff(Staff.builder().firstName("John").lastName("Smith").build())
+
+  private fun stubRepositoryCall() {
+    whenever(offenderIdentifierRepository.findOffenderIdentifiersByOffender_NomsId(ArgumentMatchers.anyString())).thenReturn(
+      listOf(
+        OffenderIdentifier().also {
+          it.identifierType = "TYPE"
+          it.identifier = "IDENTIFIER"
+          it.issuedAuthorityText = "Comment"
+          it.issuedDate = LocalDate.parse("2024-01-01")
+          it.caseloadType = "GENERAL"
+          it.createDateTime = LocalDateTime.parse("2023-01-01T00:00:00")
+          it.rootOffenderId = 123
+          it.offender = Offender().also { offender ->
+            offender.id = 123
+            offender.rootOffender = Offender().also { rootOffender -> rootOffender.id = 123 }
+            offender.addBooking(OffenderBooking().also { booking -> booking.bookingId = 1001 })
+            offender.nomsId = "A1234BC"
+          }
+        },
+
+        OffenderIdentifier().also {
```

Comment:
Again, probably better with `apply` here. It's difficult because you read the [documentation](https://kotlinlang.org/docs/scope-functions.html#run) and it's full of technical jargon but doesn't really explain the intentions behind each scope function.